.github/containerscan/allowedlist.yaml

```diff
@@ -1,9 +0,0 @@
-general:
```

Comment:
Do we have a reason to remove this file? I don't really know what it's for, just checking we didn't remove something used.

Reply:
we don't need this configuration anymore, 
I replaced the deprecated action scan-container https://github.com/Azure/container-scan
by
https://github.com/aquasecurity/trivy-action